BangazonAPI/TestBangazonAPI/DatabaseFixture.cs

```diff
@@ -1,12 +1,83 @@
-﻿using System;
+﻿using BangazonAPI.Models;
+using Microsoft.Extensions.Configuration;
+using System;
 using System.Collections.Generic;
-using System.Linq;
+using System.Data.SqlClient;
 using System.Text;
-using System.Threading.Tasks;
 
 namespace TestBangazonAPI
 {
-    class DatabaseFixture
+    public class DatabaseFixture : IDisposable
     {
+        private readonly string ConnectionString = @$"Server=localhost\SQLEXPRESS01;Database=BangazonAPI;Trusted_Connection=True;";
```

Comment:
Same here-- if both of your teammates are running on `SQLEXPRESS` and you're the only one on `01`, change before you merge and that way only one of you has to change things around.  If they have numbers attached to theirs, ignore me.